### PR TITLE
docs: add botwayce deploy docs

### PR DESCRIPTION
**Pull request by sz9751210**

### CODE REVIEW

File: docs/pages/marketplace/botwayce.en-US.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Zeabur provides a one-click deployment feature for the Botway CE service, enabling you to swiftly set up Botway CE on Zeabur.
+
+## Deploying Botway CE Service
+
+In your project, click **Create Service**, select **Marketplace** and then search for `Botway CE`.
+
+![Deploy "Botway CE"](/marketplace/botwayce/deploy-botwayce.gif)
+
+After selecting「Botway CE」, Zeabur will automatically deploy the Botway CE service for you.
```
Comment (pan93412) [inline docs/pages/marketplace/botwayce.en-US.mdx:17]:
```suggestion
After selecting "Botway CE," Zeabur will automatically deploy the Botway CE service for you.
```

File: docs/pages/marketplace/botwayce.zh-TW.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- `PASSWORD`
+- `PKGS`
+
+## 連接 Botway CE
+
+我這邊使用 Zeabur 內建的 Generate Domain 功能建立 Domain, 在 **Networking** 的部分點擊 Generate Domain。
+
+![建立 "Botway CE Domain"](/marketplace/botwayce/gen-botwayce-domain.gif)
+
+當建立好後打開 Domain 並輸入 Variable 的密碼即可開始使用 Botway CE
```
Comment (pan93412) [inline docs/pages/marketplace/botwayce.zh-TW.mdx:34]:
```suggestion
當建立好後打開 Domain 並輸入 `PASSWORD` 環境變數內的密碼即可開始使用 Botway CE。
```

**Review by pan93412 — CHANGES_REQUESTED** [review]
You may need to port some changes to other languages.

File: docs/pages/marketplace/botwayce.zh-TW.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Zeabur 提供了一鍵部署 Botway CE 服務的功能，讓你可以快速的在 Zeabur 上部署 Botway CE。
+
+## 部署 Botway CE 服務
+
+在你的專案中，點擊 **建立服務**，選擇 **Marketplace** 之後搜尋 `Botway CE`。
+
+![部署 "Botway CE"](/marketplace/botwayce/deploy-botwayce.gif)
+
+點選 "Botway CE" 後，Zeabur 會自動幫你部署 Botway CE 服務。
```
Comment (pan93412) [inline docs/pages/marketplace/botwayce.zh-TW.mdx:17]:
```suggestion
點選「Botway CE」後，Zeabur 會自動幫你部署 Botway CE 服務。
```

（你中文的引號是對的）

File: docs/pages/marketplace/botwayce.zh-CN.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Zeabur 提供了一键部署 Botway CE 服务的功能，使你能够迅速在 Zeabur 上布置 Botway CE。
+
+## 部署 Botway CE 服务
+
+在你的项目中，点击 **建立服务**，选择 **Marketplace** 然后搜索 `Botway CE`。
+
+![部署 "Botway CE"](/marketplace/botwayce/deploy-botwayce.gif)
+
+选择 "Botway CE" 后，Zeabur 会自动帮你完成 Botway CE 服务的部署。
```
Comment (pan93412) [inline docs/pages/marketplace/botwayce.zh-CN.mdx:17]:
```suggestion
选择“Botway CE”后，Zeabur 会自动帮你完成 Botway CE 服务的部署。
```

File: docs/pages/marketplace/botwayce.ja-JP.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+ZeaburはBotway CEサービスのワンクリックデプロイ機能を提供しており、Zeabur上で迅速にBotway CEをセットアップできます。
+
+## Botway CEサービスのデプロイ
+
+プロジェクト内で **Create Service** をクリックし、**Marketplace** を選択してからBotway CEを検索してください。
+
+![Botway CEをデプロイする](/marketplace/botwayce/deploy-botwayce.gif)
+
+"Botway CE" を選択すると、Zeaburが自動的にBotway CEサービスをデプロイします。
```
Comment (pan93412) [inline docs/pages/marketplace/botwayce.ja-JP.mdx:17]:
```suggestion
「Botway CE」を選択すると、Zeaburが自動的にBotway CEサービスをデプロイします。
```

**Review by pan93412 — APPROVED** [review]

**Review by MichaelYuhe — APPROVED** [review]

**Comment by MichaelYuhe** [comment]
LGTM. Thanks for the contributions

**Comment by sz9751210** [comment]
> LGTM. Thanks for the contributions

I am very fond of your products and hope to gradually understand all of your features while I am writing documents.

**Pull request merged**